### ISSUE TITLE
Support for blind or otherwise disabled users

### ISSUE THREAD

**Issue by cwille97** [mbi/django-simple-captcha#199]
There should be support for accessibility features that allow blind or otherwise disabled users to fill out the captcha in a reasonably accurate manner.

**Comment by mbi** [comment]
Well, for starters this app...

> Supports text-to-speech audio output of the challenge text, for improved accessibility

What others improvements would you suggest could improve accessibility, without compromising security?

**Comment by cwille97** [comment]
@mbi  My mistake. I'll go ahead and close this Issue, I did a search on the readthedocs and couldn't find anything regarding disability support, but I see it on the index page now.
